src/server/game/BattlePets/BattlePetMgr.h

```diff
@@ -69,6 +74,15 @@ enum class BattlePetError : uint8
     TooHighLevelToUncage   = 7  // This pet is too high level for you to uncage.
 };
 
+enum class BattlePetSlot : uint8
+{
+    Slot0 = 0,
+    Slot1 = 1,
+    Slot2 = 2,
+
+    Count
```

Comment:
```suggestion
    Max
```
^ would be a better name I guess?

Reply:
I've used the same name as the enum classes in DBCEnums.h (https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/DataStores/DBCEnums.h#L292). I don't know if there are any criteria for choosing this name

Reply:
Well imo it's the last entry, which is also `Max`. Let's see what the others say.